README.md

```diff
@@ -62,23 +62,28 @@ Additional requirements for Apple:
 * XCode 11+ ([download](https://developer.apple.com/xcode/))
 * XCode command line tools: `xcode-select --install`
 
+
 To open the macOS project on XCode:
 ```
-open ./platforms/ios/Outline.xcodeproj
+open ./platforms/osx/Outline.xcodeproj
 ```
 
 To open the iOS project on XCode:
 ```
-open ./platforms/osx/Outline.xcodeproj
+open ./platforms/ios/Outline.xcodeproj
 ```
 
 To build for macOS (OS X), run:
 
```

Comment:
Remove the extra line here and below.

Reply:
You don't like the code block, you mean?